CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.3.1
+
+## @rjsf/fluent-ui
+
+- Fix RadioWidget's onChange return value.
+
```

Comment:
Since we haven't release 5.3.0 yet, please move this underneath that section, thanks!